src/main/client/src/App.tsx

```diff
@@ -45,6 +48,11 @@ export function App() {
                 <Route index element={<AchievementList />} />
               </Route>
               {/*QR*/}
+              <Route path="qr-scanned" element={<QRScanResult />}></Route>
```

Comment:
Move this inside the QR route as /qr/szkennelt. Alos use ungarische route names IMO.

Reply:
This path comes from the backend, because it redirects to this page with the appropriate results as query when someone manually redeems the code. Contact Schámi if you another route.